src/content/docs/pt-br/guides/integrations-guide/tailwind.mdx

```diff
@@ -0,0 +1,231 @@
+
+### Instalação Rápida
+
+O comando `astro add` automatiza a instalação para você. Execute um dos seguintes comandos em uma nova janela do terminal. (Se você não tem certeza de qual gerenciador de pacotes está usando, execute o primeiro comando.) Em seguida, siga as instruções e digite "y" no terminal (significando "sim") para cada um.
+
+```sh
+# Usando NPM
+npx astro add tailwind
+# Usando Yarn
+yarn astro add tailwind
+# Usando PNPM
+pnpm astro add tailwind
+```
+
+Se encontrar algum problema, [sinta-se à vontade para relatar no GitHub](https://github.com/withastro/astro/issues) e tente as etapas de instalação manual abaixo.
+
+### Manual Install
+
+Primeiro, instale os pacotes `@astrojs/tailwind` e `tailwindcss` usando seu gerenciador de pacotes. Se você estiver usando npm ou não tiver certeza, execute isso no terminal:
+
+```sh
+npm install @astrojs/tailwind tailwindcss
+```
+
+Então, adicione essa integração em seu arquivo `astro.config.*` dentro da propriedade `integrations`:
+
+```diff lang="js" "tailwind()"
+  // astro.config.mjs
+  import { defineConfig } from 'astro/config';
++ import tailwind from '@astrojs/tailwind';
+
+  export default defineConfig({
+    // ...
+    integrations: [tailwind()],
+    //             ^^^^^^^^^^
+  });
+```
+
+Agora, crie o arquivo `tailwind.config.cjs` na raiz do seu projeto. Você pode usar o seguinte comando para criar uma configuração básica:
+
+```sh
+npx tailwindcss init
+```
+
+Finalmente, adicione essa configuração em seu arquivo `tailwind.config.cjs`:
+
+```diff lang="js" "content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}']"
+  // tailwind.config.cjs
+  /** @type {import('tailwindcss').Config} */
+  module.exports = {
++   content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+    theme: {
+      extend: {},
+    },
+    plugins: [],
+  };
+```
+
+## Utilização
+
+Quando você instala a integração, as classes utilitárias devem ja estar prontas para uso. Acesse a [documentação do Tailwind](https://tailwindcss.com/docs/utility-first), para aprender como usá-lo corretamente, e se achar uma classes que deseja usar, adicione ela em qualquer elemento HTML em seu projeto.
+
+O [Autoprefixer](https://github.com/postcss/autoprefixer) é automaticamente adicionado quando você está em modo desenvolvimento e em builds para produção, fazendo com que as classes do Tailwind funcionem em browsers antigos.
+
+<Video src="https://user-images.githubusercontent.com/4033662/169918388-8ed153b2-0ba0-4b24-b861-d6e1cc800b6c.mp4" type="video/mp4" />
+
+## Configuração
+
+### Configuração Tailwind
+
+Se você seguiu as instruções de Instalação Rápida e respondeu "sim" para cada prompt, verá um arquivo tailwind.config.cjs na raiz do seu projeto. Use este arquivo para fazer alterações na configuração do Tailwind. Você pode aprender como personalizar o Tailwind usando essa [documentação oficial](https://tailwindcss.com/docs/configuration).
```

Comment:
Como anteriormente "prompt" foi traduzido como "instrução", a gente pode dar uma alterada aqui para seguir isso e evitar o estrangeirismo. Também, no original há uma referência a "this file", no caso, o exemplo de código mostrando o arquivo de configuração na documentação do Tailwind e isso não foi representado aqui, fiz uma mudança para alinhar com o original.
```suggestion
Se você seguiu as instruções de Instalação Rápida e respondeu "sim" para todas, verá um arquivo `tailwind.config.cjs` na raiz do seu projeto. Use este arquivo para fazer alterações na configuração do Tailwind. Você pode aprender como personalizar o Tailwind utilizando esse arquivo na [documentação oficial](https://tailwindcss.com/docs/configuration).
```